service/controller/clusterapi/drainer.go

```diff
@@ -88,12 +79,12 @@ func NewDrainer(config DrainerConfig) (*Drainer, error) {
 	var operatorkitController *controller.Controller
 	{
 		c := controller.Config{
-			CRD:          v1alpha1.NewAWSConfigCRD(),
+			CRD:          clusterv1alpha1.NewClusterCRD(),
 			CRDClient:    crdClient,
 			Informer:     newInformer,
 			Logger:       config.Logger,
 			ResourceSets: resourceSets,
-			RESTClient:   config.G8sClient.ProviderV1alpha1().RESTClient(),
+			RESTClient:   config.CMAClient.ClusterV1alpha1().RESTClient(),
```

Comment:
The drainer needs to watch the Cluster CR for now as well. That should later be the MachineDeployment but we are not there yet.